db/migrate/20200910205330_create_logs.rb

```diff
@@ -0,0 +1,12 @@
+class CreateLogs < ActiveRecord::Migration[5.2]
+  def change
+    create_table :logs do |t|
+      t.references :user, foreign_key: true
+      t.references :symptom, foreign_key: true
+      t.string :time
```

Comment:
Same question re: string vs `DateTime`? If the idea is that we'll eventually be able to sort or filter out symptom logs by date and time, how can we set ourselves up to do that smoothly with this as a string? And how are we ensuring that the format is normalized?

Reply:
Yeah let's change it to datetime.